transport/http2_client.go

```diff
@@ -191,9 +193,11 @@ func newHTTP2Client(ctx context.Context, addr TargetInfo, opts ConnectOptions) (
 	if kp.Timeout == 0 {
 		kp.Timeout = defaultClientKeepaliveTimeout
 	}
-	icwz := int32(initialConnWindowSize)
+	dynamicWindow := true
+	icwz := int32(initialWindowSize)
 	if opts.InitialConnWindowSize >= defaultWindowSize {
 		icwz = opts.InitialConnWindowSize
+		dynamicWindow = false
```

Comment:
Can this not be false?

If I see an option called "Initial ______", I don't assume that will disable dynamic updates to ____.  Ideally there should be a *different* option to set to prevent BDP calculation from affecting the window sizes.